library/cwm/src/lib/cwm/page.rb

```diff
@@ -0,0 +1,53 @@
+module CWM
+  # A page widget is a group of widgets that are contained within it.
+  # Several pages are in turn contained within a {Pager}.
+  #
+  # {TreePager} is a {Pager}.
+  #
+  # {Tabs} is a {Pager} and a {Tab} is its {Page}
+  # (FIXME: We haven't made Tabs a subclass of Pager yet.
+  # That will come after we have TreePager working.
+  class Page < CustomWidget
+    # @return [Boolean] is this the initially selected tab
+    attr_accessor :initial
+
+    # @return [WidgetTerm] contents of the tab, can contain {AbstractWidget}s
+    attr_reader :contents
+    # @return [String] label defines name of tab header
+    attr_reader :label
+    attr_reader :widget_id
```

Comment:
I used the two attrs, label and widget_id, for the [PagerTreeItem constructor](https://github.com/yast/yast-yast2/blob/e44e8a0f15015f116a8685b4ca358217101f1db4/library/cwm/src/lib/cwm/tree_pager.rb#L14).